src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/AmendApplicationContestedAboutToStartHandler.java

```diff
@@ -31,6 +34,10 @@ public GenericAboutToStartOrSubmitCallbackResponse<Map<String, Object>> handle(C
                                                                                    String userAuthorisation) {
         service.defaultCivilPartnershipField(callbackRequest);
         service.defaultTypeOfApplication(callbackRequest);
+        Map<String, Object> caseData = callbackRequest.getCaseDetails().getData();
```

Comment:
not covered in test

coverage is low 
https://sonarcloud.io/component_measures?metric=new_coverage&selected=finrem-case-orchestration%3Asrc%2Fmain%2Fjava%2Fuk%2Fgov%2Fhmcts%2Freform%2Ffinrem%2Fcaseorchestration%2Fhandler%2FAmendApplicationContestedAboutToStartHandler.java&view=list&pullRequest=844&id=finrem-case-orchestration

Reply:
Ok, thanks Rishi, I'll take a look